main/cloudfoundry_client/client.py

```diff
@@ -42,6 +43,7 @@ def __init__(self, target_endpoint, credential_manager):
         self.service_instances = ServiceInstanceManager(target_endpoint, credential_manager)
         self.service_keys = ServiceKeyManager(target_endpoint, credential_manager)
         self.service_plans = ServicePlanManager(target_endpoint, credential_manager)
+        self.users = ServicePlanManager(target_endpoint, credential_manager)
```

Comment:
I think you did not instanciate the good manager

Reply:
Fixed the below issue. Using the UserManager
```suggestion
        self.users = ServicePlanManager(target_endpoint, credential_manager)
```